feature_extraction.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, **kwargs):
         self.encoder_class = ce.TargetEncoder
 
     def fit(self, X, y, **kwargs):
+        y = to_numpy_label_inputs([y])
```

Comment:
@pknut this mixes up the logic of this particular transformer with the formatting logic of adapter. It should be moved to a seperate adapter-transformer that performes this operation on y and passes it forward.